flow/connectors/utils/env.go

```diff
@@ -45,3 +45,23 @@ func GetEnvInt(name string, defaultValue int) int {
 
 	return i
 }
+
+// GetEnvString returns the value of the environment variable with the given
+// name or defaultValue if the environment variable is not set.
+func GetEnvString(name string, defaultValue string) string {
+	val, ok := GetEnv(name)
+	if !ok {
+		return defaultValue
+	}
+
+	return val
+}
+
+func IsTestEnv() bool {
```

Comment:
This is generally pretty dangerous unless used carefully. Since it means you aren't testing when `!IsTestEnv` & that's the case that matters

Reply:
100% agreed - filed: https://github.com/PeerDB-io/peerdb/issues/672 and will follow-up with fixing the tests. The urgency in landing the PR is for a ongoing issue.